content/en/about/board/_index.md

```diff
@@ -82,14 +82,6 @@ Previously, Russ founded and led the Developer Collaboration effort at Nike.
 Russ's drive and passion is to enable all software engineers to achieve incredible technical and business throughput via quality tooling and streamlined work process.
 {{< /board-member >}}
 
-{{< board-member name="Silona Bonewald" title="Treasurer" image="/images/about/Silona_Bonewald.jpeg" >}}
```

Comment:
Should we move Silone and max to the "Past Directors and Officers"?
I don't know if all translations have this section but EN does.

Reply:
Yes we should, good spot @spier!

Reply:
Spotted a few other issues across languages. Fixed what I can and raised https://github.com/InnerSourceCommons/innersourcecommons.org/issues/705 to address the rest.